devtools/src/main/scala/kalix/devtools/impl/DockerComposeUtils.scala

```diff
@@ -61,6 +73,8 @@ case class DockerComposeUtils(file: String, envVar: Map[String, String]) {
       Seq.empty
     }
 
+  // FIXME: process output is being printed in sbt console as error
+  // note to self: this is seems to be similar to sbt native packager printing errors when build docker images
```

Comment:
I still need to understand why this happens.

Reply:
I did some more tests on this. Here is what I observed. 

When running `docker-compose up` directly, we don't have any error output. At least not on my shell. 

```shell
=❯ docker-compose up
[+] Running 1/0
 ✔ Container scala-protobuf-fibonacci-action-kalix-proxy-1  Created                                                                                                    0.0s
Attaching to scala-protobuf-fibonacci-action-kalix-proxy-1
scala-protobuf-fibonacci-action-kalix-proxy-1  | Picked up JAVA_TOOL_OPTIONS: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml
...
^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...
[+] Running 1/1
 ✔ Container scala-protobuf-fibonacci-action-kalix-proxy-1  Stopped                                                                                                    0.6s
canceled
```

When running with the current version (v1.2.1), we also don't get any error logger. 
In the current version, `DockerComposeUtils` runs in the same process as the plugin.

```shell
=❯ sbt runAll
...
 Container scala-protobuf-fibonacci-action-kalix-proxy-1  Created
Attaching to scala-protobuf-fibonacci-action-kalix-proxy-1
scala-protobuf-fibonacci-action-kalix-proxy-1  | Picked up JAVA_TOOL_OPTIONS: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml
...
^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...
 Container scala-protobuf-fibonacci-action-kalix-proxy-1  Stopping
```

But, when running with the code in this PR, we get:

```shell
[error]  Container scala-protobuf-fibonacci-action-kalix-proxy-1  Created
[info] Attaching to scala-protobuf-fibonacci-action-kalix-proxy-1
[error] scala-protobuf-fibonacci-action-kalix-proxy-1  | Picked up JAVA_TOOL_OPTIONS: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml
[error] scala-protobuf-fibonacci-action-kalix-proxy-1  |
[info] scala-protobuf-fibonacci-action-kalix-proxy-1  | 15:56:38.892 INFO  kalix.proxy.KalixProxyMain - Starting Kalix Proxy version [1.1.13] Dev Mode
...
^C[error] Gracefully stopping... (press Ctrl+C again to force)
[error] Aborting on container exit...
[info] shutting down sbt server
[error]  Container scala-protobuf-fibonacci-action-kalix-proxy-1  Stopping
```

Here the behaviour is quite different. First, the lines get a log-level tag (`[info]` and `[error]`). And some of the lines are printed as `error` others as `info`. 

In this run, `DockerComposeUtils` runs from inside the `KalixRunner` and is in a forked process. 

The process we use to spawn docker-compose from inside the JVM seems to be emitting some of the logs to stderr. Then sbt runs a forked process, it captures everything that is emitted to stdout and stderr and log it with the log level as above.

I did a quick test with this:

```scala
object Hello extends  App {
  System.out.println("this goes to stdout")
  System.err.println("this goes to stderr")
}
```

When run with `run/fork := true`, we get:

```scala
[error] this goes to stderr
[info] this goes to stdout
```

When run with `run/fork := false`, we get:

```scala
this goes to stderr
this goes to stdout
```

So, first question is, why docker-compose is sending this output to stderr? 
We could try to forward this output to stdout, but then in case of a real error, sbt will print it as `info`.

Reply:
Sorry for the wall of text. Now I realise how large it became. 
Anyway, it's useful to me to organise my thoughts and register somewhere.